js/index.js

```diff
@@ -38,5 +38,122 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
+// -------Header Section------ //
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navItem = document.getElementsByTagName('a');
+navItem[0].textContent = siteContent["nav"]["nav-item-1"]
+navItem[1].textContent = siteContent["nav"]["nav-item-2"]
+navItem[2].textContent = siteContent["nav"]["nav-item-3"]
+navItem[3].textContent = siteContent["nav"]["nav-item-4"]
+navItem[4].textContent = siteContent["nav"]["nav-item-5"]
+navItem[5].textContent = siteContent["nav"]["nav-item-6"]
+
+Array.from(navItem).forEach(links => {
+  links.style.color = "green";
+})
```

Comment:
This is great. Here's another approach: 
`const nav1 = document.querySelectorAll('nav a');
nav1.forEach((value, index) => {
  value.innerHTML = siteContent.nav[`nav-item-${index + 1}`]
});
nav1.forEach(param => param.style.color = 'green');
`